docs/multicluster/getting-started.md

```diff
@@ -0,0 +1,417 @@
+# Antrea Multi-Cluster Controller Installation
+
+## Prepare Antrea Multi-Cluster Image
+
+For Antrea multi-cluster, there will be only one image `antrea/antrea-multicluster-controller:latest`
+for all controllers, you can let Kubernetes to pull this image from Docker Hub
```

Comment:
We just have one controller? If so, remove "for all controllers".

BTW, I feel we could just name the image "antrea/antrea-multicluster".

Reply:
to pull -> pull

It is not really K8s pulls the image, so maybe just say "you can pull the image from Docker Hub".

Reply:
due to the image name has been updated to `antrea/antrea-mc-controller:latest` in the manifests, I updated the doc to match the manifests, let me know if you have any other opinion, thanks.